controllers/csiaddonsnode_controller.go

```diff
@@ -18,19 +18,30 @@ package controllers
 
 import (
 	"context"
+	"fmt"
 
+	csiaddonsv1alpha1 "github.com/csi-addons/kubernetes-csi-addons/api/v1alpha1"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/connection"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/util"
+	"github.com/go-logr/logr"
+
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
 
-	csiaddonsv1alpha1 "github.com/csi-addons/kubernetes-csi-addons/api/v1alpha1"
+const (
+	csiAddonsNodeFinalizer = "csiaddons.openshift.io/csiaddonsnode"
```

Comment:
use `GroupVersion.Group+"csiaddonsNode"` to avoid duplicate of `csiaddons.openshift.io` if something changes in future it we don't need to changes things in many places.